figma-shop/src/app/pages/product-details/product-details.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Product } from 'src/app/shared/models/product.model';
+import { ProductServiceService } from '../../shared/services/product-service.service';
+
+@Component({
+  selector: 'app-product-details',
+  templateUrl: './product-details.component.html',
+  styleUrls: ['./product-details.component.scss']
+})
+export class ProductDetailsComponent implements OnInit {
+  @Input() product: any;
+  constructor(private productService: ProductServiceService) { }
+
+  ngOnInit(): void {
+  }
+
+  updateActive(isActive: boolean) {
```

Comment:
Always write modificators for methods and fields in your classes:
![image](https://user-images.githubusercontent.com/62940283/100315806-edd0a680-2fc1-11eb-9d01-7cda3979fa3a.png)